iglu-registry/example/docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: '3'
+services:
+    iglu-registry:
+        container_name: iglu-registry
+        image: iglu-registry
+        command: [ "--config", "/snowplow/config/application.conf" ]
+        stdin_open: true
+        tty: true
+        depends_on:
+         - postgres
+        ports:
+         - "8080:8080"
+        volumes:
+         - "./config/application.conf:/snowplow/config/application.conf"
+        logging:
+          options:
+            max-size: 1G
+        environment:
+          - API_KEY=1d9c7e70-012b-11e8-ba89-0ed5f89f718b
+    postgres:
```

Comment:
Can we put postgres first since iglu-registry depends on it?

Reply:
Sure.